server/cluster/cluster.go

```diff
@@ -2169,8 +2169,12 @@ func (c *RaftCluster) PutMetaCluster(meta *metapb.Cluster) error {
 }
 
 // GetRegionStatsByRange returns region statistics from cluster.
-func (c *RaftCluster) GetRegionStatsByRange(startKey, endKey []byte) *statistics.RegionStats {
-	return statistics.GetRegionStats(c.ScanRegions(startKey, endKey, -1))
+// if useHotFlow is true, the hot region statistics will be returned.
```

Comment:
Why do we need it?

Reply:
the new sql grama "show table t1 partition(p0,p1)" will use it.

Reply:
Is this comment written in the wrong place?